__docs__/wonder-blocks-tokens/semantic-color.stories.tsx

```diff
@@ -0,0 +1,106 @@
+import {StyleSheet} from "aphrodite";
+import * as React from "react";
+import {semanticColor, sizing} from "@khanacademy/wonder-blocks-tokens";
+import {ActionColorGroup} from "../components/color";
+import {View} from "@khanacademy/wonder-blocks-core";
+import {Heading} from "@khanacademy/wonder-blocks-typography";
+
+export default {
+    title: "Packages/Tokens/Testing/SemanticColors",
+    parameters: {
+        chromatic: {
+            disableSnapshot: true,
+        },
+    },
+};
+
+const ColorGroupStory = ({
+    category,
+    group,
+}: {
+    category: Record<string, Record<string, string>>;
+    group: string;
+}) => {
+    return (
+        <View style={styles.gridCompact}>
+            <ActionColorGroup
+                category={category}
+                group={group}
+                includeExample={false}
+            />
+        </View>
+    );
+};
+
+export const SemanticColors = {
```

Comment:
This story puts everything together too. Enabled chromatic for this story so we can have visual regression tests for when semantic colors change (I believe we don't have any test coverage on this right now, other than when it changes and it's used in other components)!